content/en/05-use-case-tutorials/network-monitoring.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Network Monitoring"
+metaTitle: "Using Pixie | Use Cases | Network Monitoring"
+metaDescription: "Network monitoring using Pixie."
+order: 1
+---
+
+This tutorial will demonstrate how to use Pixie to:
+
+- [Visualize the flow of network traffic within your cluster.](#visualize-network-traffic-flowing-within-the-cluster)
+- [Visualize the flow of DNS requests within your cluster.](#visualize-dns-requests-made-within-the-cluster)
+- [Visualize TCP drops and TCP retransmits across your cluster.](#visualize-tcp-drops-and-tcp-retransmissions-across-the-cluster)
+
+**Prerequisites**
+
+1. You will need a Kubernetes cluster with Pixie installed. If you do not have a cluster, you can create a minikube cluster and install Pixie using our [install guides](/installing-pixie/quick-start/).
```

Comment:
heads up, this will have to be updated when my install guide changes land. you may want to link /installing-pixie and call it "installation steps"